jsx/lib/localize-instrument.js

```diff
@@ -0,0 +1,41 @@
+function localizeInstrument(rawInstrument, lang = 'en-ca') {
```

Comment:
This file seems to be just js, not jsx.

Reply:
That's true but where do we put it? a `/js` folder? or `/htdocs/js`?

FWIW I agree with this commenter and think we shouldn't differentiate between `jsx` and `js` via folder names or file extensions.
https://github.com/airbnb/javascript/pull/985#issuecomment-239145468